react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,40 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is React?
+It's a JS library that we can utilize to make web interfaces
+- What is a React component?
+A componenet is a piece of of the user interface that has its own appearnce and logc, in JS they are functions are return markup
+- How do you mount a React app (root component) to the DOM?
+with the export default keywords followed by the component
```

Comment:
Not quite! Look at `index.js` and see how we're injecting our React application into the DOM